server/container_create_linux.go

```diff
@@ -714,7 +714,7 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		mountLabel,
 		containerInfo.RunDir,
 		s.config.DefaultMountsFile,
-		containerInfo.RunDir,
+		containerCwd,
```

Comment:
https://github.com/cri-o/cri-o/blob/4167506e8da571df0dda8543e6b89159ef36fc9a/server/container_create_linux.go#L761 Let's set this to `mountPoint`